### PR TITLE
fix minor typo

### PR DESCRIPTION
**Pull request by hchenxa**
**What this PR does / why we need it**:
Typo error
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/cloudprovider/providers/gce/gce_loadbalancer.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func init() {
 		panic("Incorrect default GCE L7 source ranges")
 	}
 
-	flag.Var(&lbSrcRngsFlag, "cloud-provider-gce-lb-src-cidrs", "CIDRS opened in GCE firewall for LB traffic proxy & health checks")
+	flag.Var(&lbSrcRngsFlag, "cloud-provider-gce-lb-src-cidrs", "CIDRs opened in GCE firewall for LB traffic proxy & health checks")
```
Comment (xiangpengzhao) [inline pkg/cloudprovider/providers/gce/gce_loadbalancer.go]: sgtm. `CIDRS` in L79 also can be changed like this.

Comment (hchenxa) [inline pkg/cloudprovider/providers/gce/gce_loadbalancer.go]: good catch, updated

**Pull request merged**